static/js/browser2/out_arcs.tsx

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import axios from "axios";
+import _ from "lodash";
+
+const ignoredOutArcProperties = new Set([
+  "provenance",
+  "kmlCoordinates",
+  "geoJsonCoordinates",
+  "geoJsonCoordinatesDP1",
+  "geoJsonCoordinatesDP2",
+  "geoJsonCoordinatesDP3",
+]);
+const maxNameWithLanguageValues = 10;
+
+interface OutArcsSectionPropType {
+  dcid: string;
+  outArcLabels: Array<string>;
+  provDomain: { [key: string]: URL };
+}
+
+interface OutArcsSectionStateType {
```

Comment:
nit: can call this OutArcSection and ditto for InArcSection

Reply:
do you mean just remove the plural? Because I don't think I can remove the "StateType" because then it would be the same name as the component

Reply:
yep, sorry for confusion, should be "OutArcSectionStateType"